src/scancode/cli.py

```diff
@@ -579,10 +583,24 @@ def scan_one(input_file, scanners, diag=False):
     return scan_result
 
 
-def save_results(files_count, scanned_files, format, input, output_file):
+def has_findings(active_scans, file_data):
+    for scan_name in active_scans:
+        if file_data.get(scan_name):
+            return True
+
+    return False
+
+
+def save_results(scanners, only_findings, files_count, scanned_files, format, input, output_file):
     """
     Save scan results to file or screen.
     """
+
+    if only_findings:
+        active_scans = [k for k, v in scanners.items() if v[0] and v[1]]
```

Comment:
what are v[0] and v[1] ? I find it is usually best to avoid indexed access and use names instead, it is less brittle and clearer.